src/lib/RoomParticipant.ts

```diff
@@ -79,7 +79,10 @@ export class RemoteParticipant extends RoomParticipant {
     }
 }
 
-export type RemoteParticipantState = Omit<RemoteParticipant, "updateStreamState">;
+export type RemoteParticipantState = Omit<
```

Comment:
though with the definition of the state type here, maybe it is something the RoomConnection handles... I'm not fussed either way really just thought I'd share my thoughts 😄

Reply:
Agree this should be exported from the useRoomConnection module